core/templates/pages/exploration-player-page/layout-directives/audio-bar.component.spec.ts

```diff
@@ -118,7 +111,7 @@ describe('Audio Bar Component', () => {
       value: 100
     };
     let currentTimeSpy = spyOn(
-      audioPlayerService, 'setCurrentTime').and.returnValue(null);
+      audioPlayerService, 'setCurrentTime').and.returnValue();
```

Comment:
Please replace all occurences of `.and.returnValue()` with `.callThrough()` in this PR

Reply:
Done